doc/doxygen/src/driver-guide.md

```diff
@@ -266,18 +266,30 @@ int16_t tmpabc_read(const tmpabc_t *dev)
 }
 @endcode
 
-Whenever you implement status/error return values in your driver, they
-should be named, meaning that the driver MUST define an enum assigning names to
-the actual used value, e.g.
+Whenever you implement status/error return values in your driver, magic numbers
+must ***not*** be used. Instead, use negative `errno` codes (such as `-EIO` for
```

Comment:
```suggestion
MUST ***NOT*** be used. Instead, use negative `errno` codes (such as `-EIO` for
```

I personally like using caps for this, kind of like RFC, but I don't insist on this.